lexicons/com/atproto/account/delete.json

```diff
@@ -4,7 +4,20 @@
   "defs": {
     "main": {
       "type": "procedure",
-      "description": "Delete an account."
+      "description": "Delete a user account with a token and password.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["handle", "password",  "token"],
```

Comment:
Check out #492— should we switch `handle` to `identifier` here to mirror it?

Reply:
hmmm probably...

altho actually, maybe it makes more sense to do `did`. to impress on the caller that you're deleting, not just the PDS concept of the user but the repo itself as well 🤔

Reply:
btw ended up switching out for `did` for the above reason^^ 
As well, I think generally, we should only use `handle` as an "entry point" (login/password reset/discovery) & did for everything else. 

https://github.com/bluesky-social/atproto/pull/488/commits/2125fa4ca700bd62068a1f5ee79169cb646f3238